packages/gatsby-theme-docs/src/layouts/internals/layout-release-note.js

```diff
@@ -1,54 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { MDXRenderer } from 'gatsby-plugin-mdx';
-import styled from '@emotion/styled';
-import { ThemeProvider as UiKitThemeProvider } from 'emotion-theming';
+import { css } from '@emotion/core';
 import SpacingsStack from '@commercetools-uikit/spacings-stack';
-import SpacingsInline from '@commercetools-uikit/spacings-inline';
-import Stamp from '@commercetools-uikit/stamp';
 import { Markdown, designSystem } from '@commercetools-docs/ui-kit';
 
-const DateElement = styled.div`
-  line-height: ${designSystem.typography.lineHeights.small};
-`;
-const Topics = styled.div`
-  color: ${designSystem.colors.light.textInfo};
-  font-size: ${designSystem.typography.fontSizes.small};
+import LayoutReleaseNoteBody from './layout-release-note-body';
+import Link from '../../components/link';
+
+const linkStyles = css`
+  text-decoration: none;
+  color: ${designSystem.colors.light.textPrimary} !important;
 
-  > * + * {
-    padding-left: ${designSystem.dimensions.spacings.xs};
-    margin-left: ${designSystem.dimensions.spacings.xs};
-    border-left: 1px solid ${designSystem.colors.light.surfaceSecondary3};
+  :hover {
+    color: ${designSystem.colors.light.linkNavigation} !important;
   }
 `;
-const stampTheme = {
-  fontSizeDefault: designSystem.typography.fontSizes.extraSmall,
-  // Override the `critical` style which is used for the "fix" type
-  colorError95: designSystem.colors.light.surfaceForReleaseNoteTypeFix,
-  colorError: designSystem.colors.light.borderForReleaseNoteTypeFix,
-};
 
 const ReleaseNote = (props) => {
   return (
     <SpacingsStack scale="m">
-      <Markdown.H3>{props.title}</Markdown.H3>
-      <DateElement>{props.date}</DateElement>
-      <UiKitThemeProvider theme={stampTheme}>
-        <SpacingsInline>
-          <Stamp tone={mapTypeToTone(props)}>{mapTypeToLabel(props)}</Stamp>
-        </SpacingsInline>
-      </UiKitThemeProvider>
-      {props.topics.length > 0 && (
-        <Topics>
-          {props.topics.map((topic) => (
-            <span key={topic}>{topic}</span>
-          ))}
-        </Topics>
-      )}
-
-      <div>
-        <MDXRenderer>{props.body}</MDXRenderer>
-      </div>
+      <Markdown.H3
+        css={css`
+          font-weight: ${designSystem.typography.fontWeights.bold};
+        `}
```

Comment:
Hmm I'm not super happy with this overwrite. Why do we need that? It's kind of inconsistent. cc @Shecki

Reply:
we also just can use "< Back"

Reply:
@nkuehn please respond to Nicola's comment here.

Reply:
It's more of a question to @Shecki I think.

Reply:
i tagged @nkuehn because he identified the lack of bold style in his review.

Reply:
We'r throwing different discussions into one thread here. 

Please implement it as designed, and yes, that means that on the release notes list it's semantically H3 (and therefore should be H3 in html) but visually a different, own thing - alone because it's a clickable link. 

To me a maybe cleaner way would be to not do it as  a patch on H3 at all but provide and explicit Component of the title of a relase note in the list.